sim/core/target.go

```diff
@@ -226,8 +226,10 @@ type AttackTable struct {
 	PartialResistNatureThresholds Thresholds
 	PartialResistShadowThresholds Thresholds
 
-	DamageDealtMultiplier               float64
-	DamageTakenMultiplier               float64
+	// These damage multipliers are used if they apply to a specific attacker-defender pair only,
+	//  e.g. Stormstrike increases the caster's damage vs the debuffed mob, but noone else is affected.
```

Comment:
This whole `AttackTable` applies to a specific attacker-defender pair, so that part is irrelevant. Comment can say something like, `DamageDealt fields are applied as part of AttackerMultiplier(), and DamageTaken fields are applied in TargetMultiplier().`.